### ISSUE TITLE
Provide customisation option for Attendee Registration API

### ISSUE THREAD

**Issue by niranjan94** [fossasia/open-event-orga-server#3595]
Parent issue #3459
To be done after #3645

- ~An attendee registration implementation similar to Speakers registration~ (done in https://github.com/fossasia/open-event-orga-server/pull/3744)
- Should include custom data fields support similar to Speakers registration

More info at #2897 & #3153

**Comment by magdalenesuo** [comment]
@niranjan94 The custom-forms API can help us create customisation options for attendee API as well. But we will still need to implement additional optional fields as mentioned in #2897 . Closing this.
I'll open a new issue to implement the optional fields in Attendee API that can be added via Custom Form.
